data/core/units/monsters/Icemonax.cfg

```diff
@@ -0,0 +1,93 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Icemonax
+    name= _ "Icemonax"
+    race=monster
+    image="units/monsters/glypto/young-glypto.png"
+    [standing_anim]
+        start_time=-50
+        [frame]
+            image="units/monsters/glypto/young-glypto.png"
+            duration=50
+        [/frame]
+    [/standing_anim]
+    [standing_anim]
+        start_time=-50
+        terrain=Wo,Ww,Ww^Vm,Chs,Chw,Ss
+        [frame]
+            image="units/monsters/glypto/young-glypto.png"
+            duration=50
+        [/frame]
+    [/standing_anim]
+#    ellipse="misc/ellipse-nozoc"
+    hitpoints=35
+    movement_type=armoredfoot
+    movement=4
+    experience=26
+    level=0
+    alignment=chaotic
+    advances_to=null
+    {AMLA_DEFAULT}
+    undead_variation=mounted
+    cost=10
+    usage=fighter
+    description=_ "Icemonax are a strange beast of the frozen north, best described as a large, armored ground squirrel.  Their armor accummulates frost and snow, even in more temperate climates.  Most Icemonax are curious, often wandering into places they do not belong.  Despite this, efforts to study and domesticate them have failed, as they die soon after capture."
+    die_sound=tusker-die.ogg
+#    [movement_anim]
+#        [frame]
+#            begin=0
+#            end=150
+#            image="units/monsters/tusker/tusklet-moving.png"
+#        [/frame]
+#    [/movement_anim]
+    {DEFENSE_ANIM "units/monsters/glypto/young-glypto-defend2.png" "units/monsters/glypto/young-glypto-defend1.png" tusker-hit.ogg }
+    [defense]
+        village=50
+        frozen=50
+    [/defense]
+    [movement]
```

Comment:
This should be `movement_costs`.

The Travis job is reporting this and some other WML validation errors.